src/providers/TokenListProvider.tsx

```diff
@@ -1,8 +1,9 @@
 import { createContext, ReactNode, useCallback, useEffect, useState } from 'react';
 import { erc20Abi } from 'viem';
 import { useAccount } from 'wagmi';
+
 import { TOKEN_LIST } from '~/data/tokens';
```

Comment:
I think this could be `import { TOKEN_LIST } from '~/data';` if you add tokens path to the data/index

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/2/commits/7a49c1d0d395d1b07836e9b8b2f6b69a81758524